src/comparison.ml

```diff
@@ -7,4 +7,16 @@ let compare a b = match Pervasives.compare a b with
   | 0 -> Equal
   | -1 -> Less
   | 1 -> Greater
-  | _ -> assert false
\ No newline at end of file
+  | _ -> assert false
+
+let fromInt i = if i > 0 
```

Comment:
I don't think ti's a good idea to start an if expression at the end of the line like this. It should be on the same indent level as the branches.